src/components/AccordionItem/AccordionItem.js

```diff
@@ -138,8 +138,9 @@ export default class AccordionItem extends Component {
           {componentsX ? (
             <ChevronRight16
               className={`${prefix}--accordion__arrow`}
-              aria-label={iconDescription}
-            />
+              aria-label={iconDescription}>
```

Comment:
Do we need to double-up on `aria-label` and `title`?

Reply:
Good question @joshblack - `<title>` is for hover-over text whereas `aria-label` is for screen reader, but don't hesitate to speak up if you believe one covers another.

Reply:
@asudoh we've gotten feedback around the tooltips themselves from `title`, might be helpful for icons to default to not having it. If someone would like a `title`, they should place it on the interactive node (like `<button>`) that it's associated with.